packages/datagrid/src/components/DefaultHeaderRenderer/DefaultHeaderRenderer.component.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-
+// eslint-disable-next-line import/no-named-as-default
```

Comment:
Remove that please and fix it.

This rule is here to avoid confusion on import. It means that you have a non-default export named `QualityBar`. So when you `import QualityBar from './QualityBar.component';` it can be a mistake because there is a possibility that you wanted to `import { QualityBar } from './QualityBar.component';`

So you should never disable it.
To fix it, you can rename the QualityBar component to something like `QualityBarComponent`. So it's safe that when you `import QualityBar from './QualityBar.component';` it's clear that you don't want to import `QualityBarComponent`.

Don't forget to fix the QualityBar tests to import `{ QualityBarComponent }`instead of `{ QualityBar }`